STAR/draft-ietf-acme-star.md

```diff
@@ -418,7 +451,7 @@ new "auto-renewal" object.  The "auto-renewal" object MUST be present if the
 server supports STAR.  Its structure is as follows:
 
 - min-lifetime (required, integer): minimum acceptable value for auto-renewal lifetime, in seconds.
-- max-duration (required, integer): maximum delta between the auto-renewal end-date and start-date, in seconds.
+- max-duration (required, integer): maximum delta between the auto-renewal end-date and start-date, in seconds.  Note that this value should be comparable with the validity period of "traditional" certificates.
```

Comment:
I disagree with Ben on this one. I think operational and even commercial (cost) considerations are much more relevant here than cryptographic ones, and therefore these periods are unrelated. Besides, you can imagine CAs that don't ever issue "traditional" certs but only STAR certs.

Reply:
ok